test-suite/nthtodefault.cpp

```diff
@@ -338,51 +330,56 @@ void NthToDefaultTest::testGaussStudent() {
         NorthAmericaCorpDefaultKey(EURCurrency(), SeniorSec, Period(), 1.));
 
     ext::shared_ptr<Basket> basket(new Basket(asofDate, namesIds, 
-        std::vector<Real>(names, 100./names), thePool, 0., 1.));
+        std::vector<Real>(names, namesNotional/names), thePool, 0., 1.));
 
     ext::shared_ptr<PricingEngine> engine(
         new IntegralNtdEngine(timeUnit, yieldHandle));
 
     vector<NthToDefault> ntd;
     for (Size i = 1; i <= probabilities.size(); i++) {
         ntd.push_back (NthToDefault (basket, i, Protection::Seller, 
-            schedule, 0.0, 0.02, Actual360(), 100.*names, true));
+            schedule, 0.0, 0.02, Actual360(), namesNotional*names, true));
         ntd.back().setPricingEngine(engine);
     }
 
     QL_REQUIRE (LENGTH(hwCorrelation) == 3,
                 "correlation length does not match");
 
-    Real maxDiff = 0;
```

Comment:
Not sure I get what was going on here - why do you say this was a duplicate?

Reply:
That one is a reflow so that both test,  will have the same structure, maxDiff and diff definition on the same line.

Reply:
The Gaussian copula was tested in testGauss and in testGaussStudent

Reply:
Oh, ok, so it was duplicated elsewhere, not in this file.  Thanks.

Reply:
It's in this file but in the function above. Thanks to you